src/version.py

```diff
@@ -0,0 +1,4 @@
+major = '0'
+minor='0'
+build='alvaro'
+date='2023-06-13 21:05:44'
```

Comment:
Is this a leak of local config into the repo?

Reply:
That is normal, it changes automatically when `build.py` is executed

Reply:
Yes, also IMO that should be in `.gitignore` I didn't add it because @cpswan included it